views/client/src/components/list/list.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
-import './list.css';
+import { DragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+import update from 'immutability-helper';import './list.css';
```

Comment:
This is the WIP for drag and drop in the list.

Reply:
It is different feature than the modals isn't it? It should be another branch because it is another feature. Can you maybe just add comments to the related areas of your code?

Reply:
Yes. It's a different feature. Actually I was trying to include the PR made by Candy but the code was different from the code I wrote and I could not figure out completely, so I didn't include it and forgot to remove these import statements.

Reply:
You can merge it. I'll make another branch for other features.